### ISSUE TITLE
Failed to train model

### ISSUE THREAD

**Issue by whizz5** [acconeer/acconeer-python-exploration#64]
Unable to train model using default layers

I have loaded training data for a model and after configuring the features to inspect and the model parameters (using default layers from  "default_layers_2d.yaml" I press the "train" button. I am then presented with an alert (see attached picture) stating that training failed but I can't figure out how to resolve it. I'm aware that when using the default layers, I have to remove Conv2D and MaxPool2D as my feature frames are smaller than the size outlined in the docs. 

I will also provide the training data I used which should help trying to reproduce this issue (in the zip file)

 - Host OS: Windows
 - Acconeer software version: 2.7.1
 
Not too sure if it's a set up issue or a program bug, thanks in advance

![image](https://user-images.githubusercontent.com/52332474/103444583-60880e00-4c61-11eb-95ae-7febe327d21b.png)

[ml_session_data_2021_01_01_1728_swipe_left.zip](https://github.com/acconeer/acconeer-python-exploration/files/5759277/ml_session_data_2021_01_01_1728_swipe_left.zip)

**Comment by dj-acconeer** [comment]
Hi  whizz5,
what version of Tensorflow and Keras are you using?

**Comment by whizz5** [comment]
Update, added another data set and I was able to proceed with the training, another ticket mentioned that 2 data sets are needed

**Comment by whizz5** [comment]
> Hi whizz5,
> what version of Tensorflow and Keras are you using?

Keras version 2.4.3
Tensorflow version 2.4

**Comment by dj-acconeer** [comment]
Tensorflow 2.x support is still experimental, but I'll try to help you sort this out.

One more thing, how many labels do you use? You need at least 2 different labels.

**Comment by whizz5** [comment]
Initially, I only had one label however I added another one which resolved my issue

**Comment by whizz5** [comment]
@dj-acconeer Thank you for all the work you and your colleagues have done getting this tool up and running
